persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -96,26 +101,35 @@ public void execute() {
             errStream.println(commandUsageInfo);
             return;
         }
-
         try  {
             balProject = ProjectLoader.loadProject(Paths.get(sourcePath));
-            name = balProject.currentPackage().descriptor().org().value() + "." + balProject.currentPackage()
-                    .descriptor().name().value() + "." + "clients";
+            name = balProject.currentPackage().descriptor().name().value() + "." + "clients";
```

Comment:
project name can have both with and without the org name. So we need to handle both cases, right?

Reply:
org name is not required for the table name. But we can support it